tests/Resources/download/.htaccess

```diff
@@ -1,10 +0,0 @@
-# Apache 2.2
-<IfModule !mod_authz_core.c>
```

Comment:
Wy remove this file?

Reply:
As @matks mentioned its unused as whole tests dir is removed from production build. (Same as with htaccess im tests img folder)